src/yz_kv.erl

```diff
@@ -402,12 +402,25 @@ first_partition([{Partition, _}|_]) ->
 
 %% @private
 %%
-%% @doc Get the tree.
+%% @doc Get the tree if yz anti_entropy is enabled.
 %%
-%% NOTE: Relies on the fact that this code runs on the KV vnode
-%%       process.
 -spec get_tree(p()) -> tree() | not_registered.
 get_tree(Partition) ->
```

Comment:
Why not do the check in the get_and_set_tree, that way the common case of having a tree will just be a hit of the pdict, and the disabled case would be a pdict/appenv:get rather than always appenv:get/pdict:get